circle.yml

```diff
@@ -4,7 +4,7 @@ dependencies:
 
 machine:
   node:
-    version: 6.2.2
+    version: 8.9
```

Comment:
this shouldn't matter (anymore), right ? 

I don't really mind either way... but we just need to make sure that the client is backwards compatible with older node versions...  we should be as unopinionated as possible. 

node 4 is EOL at `2018-04-30`

Reply:
⚡️